.github/workflows/dart_ci.yaml

```diff
@@ -23,9 +25,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        sdk: [ 2.18.7, stable, beta ]
+        sdk: [ stable, beta ]
```

Comment:
Can we add 2.19.6 while we're in here?

Reply:
https://github.com/Workiva/test_html_builder/pull/46/commits/5cad8d31bbcc587ea8fc64e7cef719b80dd89ba0